build.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash -xe
```

Comment:
Maybe we should place it in tools or something?

I'm worried that having build.sh in root would be causing errors and complaints, as people will try to use it and most of the time it will fail unless they prepared their native environment.

It might be also confusing between docker/build.sh and build.sh

Reply:
Moved to docker and named build_script.sh (but maybe builder_helper.sh would be better?).

Reply:
It is ok, maybe adding a help to those scripts would help? Like you pointed to me in other PR at least some comment on top of them?